region/tourian/main/Blue Hopper Room.json

```diff
@@ -124,6 +125,7 @@
           "enemies": [["Blue Sidehopper", "Blue Sidehopper"]]
         }}
       ],
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}],
```

Comment:
Makes sense. Added a simple `unlocksDoors` on the other [1, 2] strats that can open with Supers for free, or take an extra enemy hit to open with Missiles or PB. More advanced dodging strat(s) like you mentioned would be good to add too; I imagine they'll need a detailed description, since it sounds very tricky. If I remember it's hard to control exactly how the left Hopper behaves.

We'll need to go over all the rooms in the game to refine the `unlocksDoors` strats. This PR is just intended as a conservative starting point. At the moment, the randomizer doesn't even allow putting a lock on the left door of Blue Hooper Room or Amphitheatre.